modules/@apostrophecms/settings/ui/apos/components/SettingsDisplayNamePreview.vue

```diff
@@ -0,0 +1,24 @@
+<template>
```

Comment:
Do we need this? Are you seeing misbehavior without it perhaps. We don't need to demo custom rendering of the fields just to demo it, we have the docs for that, so I thought I'd ask first.

Reply:
Ok, it was not clear if this component had to be part of the ticket. I've removed it now.